### DISCUSSION TITLE
Does landscapist support SVG?

### DISCUSSION THREAD

**Discussion by ivycatt** [skydoves/landscapist#122] [Q&A]
I have a project that have custom SVG icons, and we retrieve them from an amazon aws server with a URI. I've been struggling to find a good solution for my issue using jetpack compose.

**Comment by jocoand** [comment]
I'm using `landscapist-coil`, my workaround is by manually adding `coil-svg`
```
implementation("io.coil-kt:coil-svg:2.7.0")
```


```
val imageLoader = ImageLoader.Builder(context)
  .components {
     add(SvgDecoder.Factory())
  }
  .build()

...

CoilImage(
  imageLoader = { imageLoader },
)        

```
